README.md

```diff
@@ -81,6 +81,11 @@ flutter pub get
 * Connect a physical iOS device or select *Open iOS Simulator* from the device drop-down menu to power on Xcode's simulator.
 * Select your device from the dropdown list and hit the play button (*^R*).
 
+
+
+|     :bulb:    | There are limited resources on running iOS with different flavors, and the following solution was inspired by the  [Medium article](https://medium.com/@animeshjain/build-flavors-in-flutter-android-and-ios-with-different-firebase-projects-per-flavor-27c5c5dac10b). Basically, XCode can be configured to run with different flavors such as `Debug-dev` or `Debug-prod`, and the aforementioned article best describes the steps needed for such a configuration. Similarly, one can run the Bash script `./run-ios-flavor.bash prod` or `./run-ios-flavor.bash dev` from the root folder to run the application with a specific configuration.|
+|---------------|:---------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Imagine somebody with 0 context reading this. What are flavours? Why do I need to go to an article to understand what this paragraph means?

Keep things simple (especially here in the README), this is only meant as a quick guide on how to build the app.

For the README, I suggest only keeping something like "Running the app in debug mode is recommended for testing. To do this, run `./run-ios-flavor.bash dev`. Similarly, if you need to run the release version (not recommended), you can run `./run-ios-flavor.bash prod`."

On this note, please also add a debug recommendation below in the web section. Similarly, I suggest: "Running the app in debug mode is recommended for testing. To do this, replace `firebaseConfig.release` with `firebaseConfig.debug` in the [web/index.html](web/index.html) file."

Bottom line, we'd like people to only run the debug version when possible. On Android this is the default, but on iOS/Web for now we can't make it the default without also making changes in the CI/CD scripts (since the release executable is also built from the repo).

Please also update the corresponding section in CONTRIBUTING.md, since that doc says that running in debug mode isn't possible on iOS. You can also link the article there (not here), under something like "You can find more information about build flavours in Flutter in [this article]".